dynamoose.d.ts

```diff
@@ -306,4 +320,32 @@ declare module "dynamoose" {
     get(fn: any): any;
     set(fn: any): any;
   }
+
+  /**
+   * Transaction
+   */
+  export interface ModelTransactionConstructor<DataSchema, KeySchema> {
+    new(value?: DataSchema): ModelSchema<DataSchema>;
+    (value?: DataSchema): ModelSchema<DataSchema>;
+    readonly prototype: ModelSchema<DataSchema>;
+
+    create(item: DataSchema, options?: PutOptions, callback?: (err: Error, model: ModelSchema<DataSchema>) => void): Promise<ModelSchema<DataSchema>>;
+    create(item: DataSchema, callback?: (err: Error, model: ModelSchema<DataSchema>) => void): Promise<ModelSchema<DataSchema>>;
+    create(item: DataSchema, options?: PutOptions): Promise<ModelSchema<DataSchema>>;
+
+    get(key: KeySchema, callback?: (err: Error, data: DataSchema) => void): Promise<ModelSchema<DataSchema> | undefined>;
+
+    delete(key: KeySchema, callback?: (err: Error) => void): Promise<undefined>;
+
+    update(key: KeySchema, update: UpdateUpdate<DataSchema>, options: UpdateOption, callback: (err: Error, items: ModelSchema<DataSchema>[]) => void): void;
+    update(key: KeySchema, update: UpdateUpdate<DataSchema>, callback: (err: Error, items: ModelSchema<DataSchema>[]) => void): void;
+    update(key: KeySchema, update: UpdateUpdate<DataSchema>, options?: UpdateOption): Promise<ModelSchema<DataSchema>>;
+
+    conditionCheck(key: KeySchema, options?: ConditionOptions): void
+  }
+  export interface ConditionOptions {
```

Comment:
Does this not already exist in this file? Conditions were allowed in other methods before this.